webapp/channels/src/components/save_button.tsx

```diff
@@ -1,81 +1,68 @@
-// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
-// See LICENSE.txt for license information.
-
```

Comment:
Removing this is going to create a lint issue :sweat_smile:

Reply:
Sorryy by mistake my IDE is being quite wierd. Added it back